release/release_workflow.wdl

```diff
@@ -121,73 +68,22 @@ task do_release {
    }
 }
 
-task wait_for_published_artifact {
-    String repo
-    String version
-    
-    command <<<
-        checkIfPresent() {
-            isPresent=$(curl -s --head https://broadinstitute.jfrog.io/broadinstitute/libs-release-local/org/broadinstitute/${repo}/${version}/ | head -n 1 | grep -q "HTTP/1.[01] [23]..")
-        }
-        
-        elapsedTime=0
-        checkIfPresent
-        
-        # Allow 1 hour for the file to appear as a published artifact
-        while [ $? -ne 0 ] && [ $elapsedTime -lt 3600 ]; do
-            sleep 10;
-            let "elapsedTime+=10"
-            checkIfPresent
-        done
-        
-        exit $?
-    >>>
-    
-    output {
-        String publishedVersion = version
-    }
-}
-
-task create_update_dependency_command {
-    String dependencyName
-    String newVersion
-    String dependencyFilePath = "build.sbt"
-    
-    command {
-        echo "sed -i '' \"s/${dependencyName}[[:space:]]=.*/${dependencyName} = \\\"${newVersion}\\\"/g\" ${dependencyFilePath}"
-    }
-    
-    output {
-      String updateCommand = read_string(stdout())
-    }
-}
-
 task versionPrep {
     String organization
-    String repo
-    String file
-    String regexPrefix
     String updateCommandTemplate
-    
-    String bash_rematch = "{BASH_REMATCH[1]}"
+    String repo = "cromwell"
```

Comment:
Is `organization` also a fixed value?

Reply:
Oh no, that's how this is tested. You clone broadinstitute/cromwell and test in your own organization. 🙂

Reply:
Ha, ok then!